blackbox_repository/conversion_scripts/scripts/nasbench201_import.py

```diff
@@ -0,0 +1,212 @@
+import bz2
+import pickle
+
+import matplotlib.pyplot as plt
+
+import pandas as pd
+import numpy as np
+from blackbox_repository.blackbox_tabular import serialize, BlackboxTabular
+import sagemaker_tune.search_space as sp
+from blackbox_repository.conversion_scripts.utils import repository_path
+from syne_tune.util import catchtime
+
+
+def str_to_list(arch_str):
+
+    node_strs = arch_str.split('+')
+    config = []
+    for i, node_str in enumerate(node_strs):
+        inputs = list(filter(lambda x: x != '', node_str.split('|')))
+        for xinput in inputs: assert len(xinput.split('~')) == 2, 'invalid input length : {:}'.format(xinput)
+        inputs = (xi.split('~') for xi in inputs )
+
+        config.extend(op for (op, IDX) in inputs)
+
+    return config
+
+
+def convert_dataset(data, dataset):
+
+    hp_cols = [f"hp_x{i}" for i in range(6)]
+
+    hps = dict()
+
+    for h in hp_cols:
+        hps[h] = []
+
+    n_hps = data['total_archs']
+
+    for i in range(n_hps):
+        config = str_to_list(data['arch2infos'][i]['200']['arch_str'])
+
+        for j, hp in enumerate(config):
+            hps[f"hp_x{j}"].append(hp)
+
+    hyperparameters = pd.DataFrame(
+        data=hps,
+        columns=hp_cols
+    )
+
+    objective_names = [
+        'valid_error',
```

Comment:
One question, before I do something wrong downstream.
In the old dataset (on our S3 bucket), we had lc_valid_epoch_X, which was accuracy * 100, and mode was 'max'.
Here, we now have validation error, and the mode is 'min' ?

Also, 'runtime' is runtime since start?

Reply:
yes that's right we now return the validation error. And yes runtime is the runtime since start